.travis.yml

```diff
@@ -16,8 +16,6 @@ matrix:
       env: TOXENV=3.5
     - python: 3.6
       env: TOXENV=3.6
-    - python: pypy
```

Comment:
Please do not remove this. We still support PyPy.
PyPy3 on Travis is super old so we don't test with it.